llvm/lib/Transforms/IPO/WholeProgramDevirt.cpp

```diff
@@ -167,6 +167,18 @@ static cl::list<std::string>
                       cl::desc("Prevent function(s) from being devirtualized"),
                       cl::Hidden, cl::CommaSeparated);
 
+/// A function is unreachable if its entry block ends with 'unreachable' IR
```

Comment:
Can you note that this is useful for removing abstract class virtual destructors, which are emitted with a trap and unreachable instruction, and add a TODO to identify these more precisely?

Reply:
related - probably update the description which currently says that this is about pure virtual functions, but most of those are already handled due to the __cxa_pure_virtual in the vtable slot.

Reply:
Updated the surrounding comment with the motivating case and a TODO for Clang's codegen, and updated PR description as suggested.